composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": ">=5.5.9 || ^7.0",
```

Comment:
Since tests no longer run on <=7.1, the minimum PHP version should be bumped as well.